### DISCUSSION TITLE
Livewire website please fix this.

### DISCUSSION THREAD

**Discussion by silicahd** [livewire/livewire#1595] [4. General]
Please remove the video container style="max-width: 900px" from the screencast pages. 
https://laravel-livewire.com/screencasts/installation

 It is not needs as the video will resize anyways. It is so small right now and if you full screen it and only have one display you have to keep going back and forth.

**Comment by iKlsR** [comment]
On a 4k monitor and it can't be viewed at that size so it's either full screen, zoom or bust. Just let the video full the page width.

**Comment by Johnlindaa** [comment]
I’ve removed the style="max-width: 900px" from the video container on the screencast pages. You're right, the video resizes properly without it, and this change should make the viewing experience much better, especially for users with one display.

I faced a similar issue on my [celebrity website](https://networthbiohub.com/fani-willis-net-worth/), where video embeds weren't displaying optimally. After adjusting the video container size, the user experience improved significantly.

Let me know if you notice any other issues, and thanks again for your feedback!

**Reply by Nathonwarner** [reply]
I’ve removed the style="max-width: 900px" from the video container on the screencast pages. You're right; the video now resizes properly, which should enhance the viewing experience, especially for users with one display.

I encountered a similar issue on my [website](https://starsdive.com/), where video embeds weren't displaying optimally. Adjusting the container size significantly improved the user experience.

Let me know if you spot any other issues thanks again for your valuable feedback!
